client/src/Components/Validator/SigMFSchema.ts

```diff
@@ -1,301 +1 @@
-            required: ['core:sample_start', 'core:sample_count'],
-            type: 'object',
-            properties: {
-              'core:comment': {
-                $id: '#/properties/annotations/items/anyOf/0/properties/core%3Acomment',
-                default: '',
-                description: 'A human-readable comment.',
-                type: 'string',
-              },
-              'core:freq_lower_edge': {
-                $id: '#/properties/annotations/items/anyOf/0/properties/core%3Afreq_lower_edge',
-                description: 'The frequency (Hz) of the lower edge of the feature described by this annotation.',
-                type: 'number',
-                minimum: -1.7976931348623157e308,
-                maximum: 1.7976931348623157e308,
-              },
-              'core:freq_upper_edge': {
-                $id: '#/properties/annotations/items/anyOf/0/properties/core%3Afreq_upper_edge',
-                description: 'The frequency (Hz) of the upper edge of the feature described by this annotation.',
-                type: 'number',
-                minimum: -1.7976931348623157e308,
-                maximum: 1.7976931348623157e308,
-              },
-              'core:generator': {
-                $id: '#/properties/annotations/items/anyOf/0/properties/core%3Agenerator',
-                description: 'Human-readable name of the entity that created this annotation.',
-                type: 'string',
-              },
-              'core:label': {
-                $id: '#/properties/annotations/items/anyOf/0/properties/core%3Alabel',
-                description: 'A short form human/machine-readable label for the annotation.',
-                type: 'string',
-              },
-              'core:sample_count': {
-                $id: '#/properties/annotations/items/anyOf/0/properties/core%3Asample_count',
-                description: 'The number of samples that this Segment applies to.',
-                type: 'integer',
-                minimum: 0,
-                maximum: 18446744073709551615,
-              },
-              'core:sample_start': {
-                $id: '#/properties/annotations/items/anyOf/0/properties/core%3Asample_start',
-                default: 0,
-                description: 'The sample index at which this Segment takes effect.',
-                minimum: 0,
-                maximum: 18446744073709551615,
-                type: 'integer',
-              },
-              'core:uuid': {
-                $id: '#/properties/annotations/items/anyOf/0/properties/core%3Auuid',
-                description: 'RFC-4122 unique identifier.',
-                format: 'uuid',
-                type: 'string',
-              },
-            },
-            additionalProperties: true,
-          },
-        ],
-      },
-    },
-  },
-  additionalProperties: false,
-};
+export const sigmfSchema = import("../,,/../../../../client/src/Components/Validator/sigmf-schema.json");
```

Comment:
Should this be ,,?

Reply:
I thought that too but it says it can't find module without it, even when I try and replace it!